servicea/src/main/java/com/example/servicea/Greeting.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This was the license used on the original obsidian example upon which this is based.